src/components/ItaliaTheme/View/Commons/Attachments.jsx

```diff
@@ -104,17 +104,10 @@ const Attachments = ({
     <RichTextSection
       tag_id={article_id}
       className="it-page-section mb-5"
-      title={
-        title ? (
-          <h2 id={`header-${article_id}`}>{title}</h2>
-        ) : (
-          <h2 id={`header-${article_id}`}>
-            {intl.formatMessage(messages.attachments)}
-          </h2>
-        )
-      }
+      title={title ? title : intl.formatMessage(messages.attachments)}
     >
       {attachments.length > 0 && attachments_view}
+      {searchResults?.[key]?.loading && !searchResults?.[key]?.loaded && <></>}
```

Comment:
Metti un commento con scritto il perché è fatto, altrimenti sembra molto strano e rischiamo di chiedercelo in futuro

Reply:
Fatto!